base/src/main/scala/com/github/j5ik2o/akka/persistence/s3/base/trace/TraceReporter.scala

```diff
@@ -5,35 +5,36 @@ import com.github.j5ik2o.akka.persistence.s3.base.config.PluginConfig
 import com.github.j5ik2o.akka.persistence.s3.base.exception.PluginException
 import com.github.j5ik2o.akka.persistence.s3.base.model.Context
 
+import scala.annotation.unused
 import scala.collection.immutable.Seq
 import scala.concurrent.Future
 import scala.util.{ Failure, Success }
 
-trait TraceReporter {
+abstract class TraceReporter(val pluginConfig: PluginConfig) {
 
-  def traceJournalAsyncWriteMessages[T](context: Context)(f: => Future[T]): Future[T]
+  def traceJournalAsyncWriteMessages[T](@unused context: Context)(f: => Future[T]): Future[T] = f
```

Comment:
Implemented to simply execute f in all methods of TraceReporter so that the user does not have to implement all methods of TraceReporter.